src/factories/request.js

```diff
@@ -70,6 +71,7 @@ class RequestFactory {
         config.client_id,
         response.access_token,
         response.expires
+        // Math.min(response.expires, config.auth.expires)
```

Comment:
@agrohs Is this required?

Reply:
This is actually related to what I believe may be a defect in the SDK, I can remove the commented out block to leave as is but love to discuss as I think the current implementation may have an issue.

Reply:
Committed removal of comment about potential issue around credential expiration.